Library/Homebrew/cask/cmd/abstract_command.rb

```diff
@@ -58,11 +58,6 @@ def self.help
         parser.generate_help_text
       end
 
-      sig { returns(String) }
-      def self.short_description
-        description[/\A[^.]*\./]
```

Comment:
sorbet couldn't find `description`, i think it was expunged in https://github.com/Homebrew/brew/pull/10397

Reply:
Probably just legacy code that can be removed. Related to https://github.com/Homebrew/brew/issues/14799.

```rb
irb(main):003:0> Cask::Cmd::Info.short_description
Traceback (most recent call last):
        4: from /usr/local/Homebrew/Library/Homebrew/brew.rb:93:in `<main>'
        3: from (irb):3
        2: from (irb):3:in `rescue in irb_binding'
        1: from /usr/local/Homebrew/Library/Homebrew/cask/cmd/abstract_command.rb:63:in `short_description'
NameError (undefined local variable or method `description' for Cask::Cmd::Info:Class)
Did you mean?  desc_info
```

Reply:
All of these files should be removed instead: https://github.com/Homebrew/brew/issues/14799